internal/commands/install.go

```diff
@@ -66,7 +66,7 @@ func installCmd(dockerCli command.Cli) *cobra.Command {
 }
 
 func runInstall(dockerCli command.Cli, appname string, opts installOptions) error {
-	muteDockerCli(dockerCli)
+	defer muteDockerCli(dockerCli)()
```

Comment:
and here ?

Reply:
actually the defer applies to the func returned by muteDockerCli (which unmutes), that's good